Yoast/javascript
# Add check for stemming exceptions to the Russian stemmer

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a check for whether the word belongs to stemming exceptions to the Russian stemmer.
PR for the stemming exceptions list: https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/184
* [Yoast SEO Premium] Improves word form recognition of Russian nouns with stems that have endings similar to noun case suffixes.

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all current specs pass
* Add more Russian nouns from the doNotStemSuffix exception list (in the morphology data file) to the specs and make sure their endings do not get removed during stemming. Examples of these words include _сипт**ом**, гр**ом**, зап**ах**_. 

* [QA] In Russian Premium: use a noun ending in "ом" or "ах" (e.g., "диплом", "агроном", "запах") as a keyphrase and use forms of those words in the title/text/meta description (e.g., "дипломами", "агроному", "запахи"). Check that the keyphrase forms are recognized by the plugin.
## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-443
